doc/tip_of_the_day.list.c

```diff
@@ -15,6 +17,11 @@ MS Money or other programs that export QIF files or OFX files. In the \
 File menu, click on the sub-menu Import and click on QIF or OFX file, \
 respectively. Then, follow the instructions provided.")
 
+ N_( "If you want to know at which directory your recent GnuCash files \
```

Comment:
**in** which directory

@jralls proper US-English?

Reply:
@fellen Yes, "in which directory".